src/docs/usage/addItem().mdx

```diff
@@ -2,34 +2,30 @@
 title: addItem()
 ---
 
-To add an item to the cart, use `addItem()`
+<div>
+
+`addItem()` adds product objects to the [cartItems](api) array. Product objects require a
+name, sku, price and currency values as shown below.
+
+```json
+{
+  "name": "Bananas",
+  "sku": "sku_GBJ2Ep8246qeeT",
+  "price": 400,
+  "image": "https://www.fillmurray.com/300/300",
+  "currency": "USD"
+}
+```
+
+Notice that we spread the product object when we pass it to addItem(). This is so that you are free to
+add other values for your own needs.
 
 ```jsx
 /**@jsx jsx */
```

Comment:
This is actually required by `theme-ui` to use the sx prop. Maybe the real solution here is to show examples without theme-ui and use native html and the style prop? I think this might be better

Reply:
Hm, I didn't know that about. @dayhaysoos I'll let you decide between `theme-ui` and native HTML. I'm currently working on the README and I'm writing it in all HTML, it might be nice to have `theme-ui` in the full docs and maybe we could provide a little message in there specifying that `use-shopping-cart` works with any UI framework or none at all.

Reply:
@ChrisBrownie55 I think I wanna use native html. I love theme-ui so much but I know that if I wasn't using theme-ui and I wanted to be able to copy/paste things from the docs, I'd be annoyed with having to install theme-ui or refactoring it myself. 

We can use different frameworks for styling and what not when I create the examples category